src/components/EuropeMap.jsx

```diff
@@ -40,7 +39,30 @@ const EuropeMap = () => {
         previousTarget = event.target;
         event.target.style =
           "fill:#29B6F6;stroke:#ffffff;stroke-width:0.11153841;stroke-miterlimit:4;stroke-dasharray:none";
-        output.innerHTML = "Selected country";
+
+        //Part that fetches country name and displays it
+        let targetID = event.target.id;
+        console.log(targetID);
+        if (targetID === "svg2") {
+          output.innerHTML = "";
+          return;
+        }
+        if (targetID.length > 2) targetID = targetID.slice(0,2);
```

Comment:
This code implements some error handling. 
First of all it checks if the target id is "svg2" and if it is then the method returns without fetching anything. (The target id will be "svg2" if you click on the ocean (not on a country)).
Secondly we check to see if the target id contains more than just an alpha2 code and if it does then we only use the first two characters of the id. 
Some countries have and id that looks like this: id="gb-gbn" - First an alpha2 code and then something else - we just need an alpha2 code. 
Do you have any suggestions to improve this error handling system? 
Should we remove this error handling system and let the backend translate the ids into something useful?

Reply:
If it works for *all* countries then it is more than OK, I actually think it is kinda nice.

Otherwise my suggestion is to manually change the `id`s on the SVG.

Reply:
We might have to manually change the <code>id</code>s on the SVG because some parts of The Great Britain have the same alpha2 code but aren't exactly the same country.